tests/UI/pages/BO/design/positions/index.js

```diff
@@ -0,0 +1,32 @@
+require('module-alias/register');
+const BOBasePage = require('@pages/BO/BObasePage');
+
+class Positions extends BOBasePage {
+  constructor() {
+    super();
+
+    this.pageTitle = 'Positions';
+
+    // Selectors
+    this.searchInput = '#hook-search';
+    this.modulePositionForm = '#module-positions-form';
+    this.searchResultHookNameSpan = `${this.modulePositionForm} section[style] header span span`;
```

Comment:
Can't found any class for or something like that ?

Reply:
Hey ! I'm the first !

Reply:
the class is the same for all hooks, the only difference is in the style or the child ID :/

Reply:
we will add a class but we will do it on develop since we do not touch twigs on 1.7.7.x